roles/addons/singularity/README.md

```diff
@@ -1,14 +1,14 @@
-# Singularity CE or SingularityPRO Ansible role
+# Singularity Ansible role
```

Comment:
All the roles in the roles directory are Ansible roles. This title is used to generate [the online documentation](http://www.bluebanquise.com/documentation/roles.html#core), *Singularity* is enough.

Reply:
done